moviebuff.go

```diff
@@ -290,3 +292,41 @@ func (m *moviebuff) GetCertifications(country string) ([]Certification, error) {
 
 	return certifications.Data, nil
 }
+
+func (m *moviebuff) GetHolidayCalendar(countryID string) (*Calendar, error) {
+	r, err := prepareRequest(m.HostURL, m.StaticToken, "/holidays/"+countryID)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := new(http.Client).Do(r)
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
+		if res.StatusCode == http.StatusForbidden {
+			return nil, ErrInvalidToken
+		}
+		if res.StatusCode == http.StatusNotFound {
+			return nil, ErrResourceDoesNotExist
+		}
+		return nil, ErrResponseNotReceived
+	}
+
+	calendarResp, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	calendarInfo := new(Calendar)
+	err = json.Unmarshal(calendarResp, calendarInfo)
+	if err != nil {
+		log.Println(string(calendarResp))
```

Comment:
There's no reason to log here, let's keep all logging on the client side. Clients can choose to use any loggers / log formats they want

Reply:
Removing it. Just logged for debugging. I forgot to remove it.